src/history/test/HistoryTests.cpp

```diff
@@ -31,6 +31,32 @@
 using namespace stellar;
 using namespace historytestutils;
 
+// Given a ledger range, compute total number of levels affected
+size_t
+numLevelsSpills(uint32_t upgradeLedger, uint32_t endLedger)
+{
+    uint32_t levels = 0;
+    uint32_t multiplier = 2;
+
+    // Given an upgrade ledger and end ledger to apply,
+    // calculate number of expected new-style merges. Note that any old-style
+    // merges in-progress complete.
+    int diff;
+    while ((diff = (endLedger / multiplier) -
```

Comment:
some explanation of this arithmetic would be good; like at least what `diff` means and why `diff>0` controls the loop and why the loop is counting instances of `diff>1`.

Reply:
fixed